android/src/main/java/com/learnium/RNDeviceInfo/RNInstallReferrerClient.java

```diff
@@ -0,0 +1,53 @@
+package com.learnium.RNDeviceInfo;
+
+import android.content.SharedPreferences;
+import android.content.Context;
+import android.os.RemoteException;
+
+import com.android.installreferrer.api.InstallReferrerClient;
+import com.android.installreferrer.api.InstallReferrerStateListener;
+import com.android.installreferrer.api.ReferrerDetails;
+
+import static com.android.installreferrer.api.InstallReferrerClient.InstallReferrerResponse;
+
+public class RNInstallReferrerClient implements InstallReferrerStateListener {
```

Comment:
Shouldn't [startConnection](https://developer.android.com/reference/com/android/installreferrer/api/InstallReferrerClient.html#startConnection(com.android.installreferrer.api.InstallReferrerStateListener)) be called somewhere? What else is the point of implementing this interface?

Reply:
You're definitely right, I freehanded a fix for it rq because I'm a bit busy at work,
I'll have more time to give this later today

Reply:
Thanks for the review @mcuelenaere !